### ISSUE TITLE
Have you finetune YOLOv5 on AGORA dataset?

### ISSUE THREAD

**Issue by linjing7** [mks0601/Hand4Whole_RELEASE#54]
Hi, thanks for your excellent work. You mention that the human detection result of AGORA test set is from YOLOv5. The detection result is rather accurate. Do you finetune YOLOv5 on AGORA training set?

**Comment by mks0601** [comment]
Sorry I can't remember clearly, but I think I fine-tuned it on AGORA training set.

**Comment by linjing7** [comment]
Okay, thank you very much.
